content/blog/entries/automate-github-for-more-than-CI CD/contents.lr

```diff
@@ -0,0 +1,144 @@
+title: Automate GitHub for more than CI/CD
+---
+categories:
+community
+cc-search
+open-source
+---
+author: zackkrida
+---
+pub_date: 2020-08-26
+---
+body:
+
+> *Get started using GitHub bots and actions for community management and repository health.*
+
+In late 2018, in the midst of being acquired by Microsoft, GitHub [launched Github Actions](https://github.blog/2018-10-16-future-of-software/) into public beta, allowing users to run code on the popular development platform for the first time. With a straightforward `YAML` configuration syntax and the power of Microsoft's Azure cloud, GitHub actions quickly rose to compete with existing Continuous Integration (CI) and Continuous Deployment (CD) platforms like **Circle CI** and **Travis CI**. GitHub actions made it easier than ever for developers to test and deploy software in the cloud, but from the beginning GitHub had bigger plans for the service.
+
+In a [2018 TechCrunch interview](https://techcrunch.com/2018/10/16/github-launches-actions-its-workflow-automation-tool/), GitHub's then head of platform acknowledged the usefulness of actions for more than CI/CD. "I see CI/CD as one narrow use case of actions. It’s so, so much more,” Lambert stressed. “And I think it’s going to revolutionize DevOps because people are now going to build best in breed deployment workflows for specific applications and frameworks, and those become the de facto standard shared on GitHub. […] It’s going to do everything we did for open source again for the DevOps space and for all those different parts of that workflow ecosystem."
+
+At Creative Commons, we use Github Actions and Bots on many of [our open-source projects](https://github.com/creativecommons?type=source) for more than CI/CD—to manage our [community team](http://localhost:5000/community/community-team/); to automate repository health; and to make tedious but frequent tasks automatic. The following examples are just a small snapshot of our existing and in-progress automations.
```

Comment:
What's the `type=source` query param for?

Reply:
I think "automate tedious but frequent tasks" is a little clearer than "make tedious but frequent tasks automatic"

Reply:
`type=source` is added by default sometimes—it hides forked projects only to show those created by a user/org